src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { Box, Sans, breakpoints } from "@artsy/palette"
+import { ViewingRoomsApp_viewingRooms } from "v2/__generated__/ViewingRoomsApp_viewingRooms.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ViewingRoomsAppProps {
+  viewingRooms: ViewingRoomsApp_viewingRooms
+}
+
+const ViewingRoomsApp: React.FC<ViewingRoomsAppProps> = props => {
+  const viewingRooms = props.viewingRooms
+
+  if (!viewingRooms?.edges?.length) {
+    return null
+  }
+
+  const viewingRoomsForLatestGrid = viewingRooms.edges
+    .map(vr => {
+      if (!vr.node) {
+        return null
+      }
+
+      if (vr.node.status != "scheduled" && vr.node.status != "live") {
+        return null
+      }
+
+      return {
+        ...vr.node,
+      }
+    })
+    .filter(vr => vr)
+
+  return (
+    <AppContainer maxWidth="100%">
+      <Box maxWidth={breakpoints.xl} mx="auto" width="100%">
+        <Sans size="10" my={3}>
+          Viewing Rooms
+        </Sans>
```

Comment:
Featured rail will go here. Not clear if we'll need a separate MP call for FeaturedRooms or we can just add `featured` to the viewing rooms call here.